index.html

```diff
@@ -39,7 +39,9 @@ <h1 class="logo">My Logo</h1>
     <section class="landing-page landing">
         <h1>I'm Silvia Glad to see you!</h1>
         <p class="prg">
-            I'm a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+            I'm a software developer! I can help you build a product ,
+             feature or website Look through some of my work and experience!
+              If you like what you see and have a project you need coded, don't hestiate to contact me.
```

Comment:
1. Please kindly add a background image in you header section. Follow design on Figma.


Your Design:
![Screenshot from 2022-05-12 18-14-41](https://user-images.githubusercontent.com/79506105/168097118-0975d58c-0866-4bb2-b839-01a1a840d77f.png)

Figma Design:

![Screenshot from 2022-05-12 18-16-14](https://user-images.githubusercontent.com/79506105/168097305-6903a098-af64-47da-b272-40ef25ea1f6e.png)

2. In Your Cards section kindly add proper spacing between Canopy, backend dev and 2015 and make sure the dots are big enough. Follow the design on figma

This is Your design:
![Screenshot from 2022-05-12 18-14-52](https://user-images.githubusercontent.com/79506105/168098004-5e8e0e5a-51d7-467a-9c1f-10187fdb6f0d.png)

Follow this design on Figma:

![Screenshot from 2022-05-12 18-16-19](https://user-images.githubusercontent.com/79506105/168098132-16c1ab50-d447-4125-bc51-4083f9c56277.png)

3. In your about section please put the LET CONNECT text in its correct postion and also adjust the drop down and the side arrow on SKills. Check Design on Figma.
Your design:
![Screenshot from 2022-05-12 18-15-10](https://user-images.githubusercontent.com/79506105/168098955-28a8da71-0bb0-4254-8ed8-cb13409d62b3.png)

Figma Template:
![Screenshot from 2022-05-12 18-16-27](https://user-images.githubusercontent.com/79506105/168099395-392ee6b4-0544-435b-badd-803cbba47c78.png)

Reply:
ok, thanks for all the helpful advice I will follow